feat(i18n): add missing translations for Japanese

siketyan opened the pull request:
## Summary

This pull request adds untranslated pages for Japanese, except auto-generated pages.

chansuke commented on src/content/docs/ja/reference/diagnostics.mdx:
```diff
@@ -0,0 +1,135 @@
+
+Biomeの診断結果には多くの情報が含まれており、通常エラーを理解して修正するために必要なすべての情報を提供します。
+
+診断はエラーだけでなく、構造的な情報、警告、チップも提供します。
```
I was thinking about the best word for 'tips'... You might consider `ヒント` or `秘訣` etc?
```suggestion
診断はエラーだけでなく、構造的な情報、警告、ヒントも提供します。
```

siketyan replied:
Nice idea, thank you. Applied in [7fc9fb7](https://github.com/biomejs/website/pull/1747/commits/7fc9fb7885eac7c7d1e89d9e9c60fa745ed1dd2b).

chansuke commented on src/content/docs/ja/reference/diagnostics.mdx:
```diff
@@ -0,0 +1,135 @@
+
+### 修正可能（Fixable）
+
+修正可能な診断はユーザによって修正可能なシチュエーションについて出力されます。これらは通常コードアクションを持つリントの診断に使われます。
```
this is just my humble opinion, but you might consider s/シチュエーション/状況/. while `シチュエーション` is meaningful, but personally, I think `修正可能なシチュエーションについて出力されます` has better readability. what do you think?

https://github.com/search?q=repo%3Abiomejs%2Fwebsite+%E7%8A%B6%E6%B3%81&type=code

siketyan replied:
Also fixed in [7fc9fb7](https://github.com/biomejs/website/pull/1747/commits/7fc9fb7885eac7c7d1e89d9e9c60fa745ed1dd2b).

chansuke reviewed (commented):
Thank you for your work!! I left some comments

siketyan commented:
I rebased this to next, and fixed format of astro.config.ts ([diff](https://github.com/biomejs/website/compare/7bc7a7c4306c486f3e97d2f0e57006af2f94f108..36b69e60f17473b93b2aaed2fd6e18d5d2d43412))
